lib/src/ConfigLoader.cc

```diff
@@ -538,9 +538,9 @@ static void loadDbClients(const Json::Value &dbClients)
                        type.begin(),
                        [](unsigned char c) { return tolower(c); });
         auto host = client.get("host", "127.0.0.1").asString();
-        auto port = client.get("port", 5432).asUInt();
+        unsigned short port = client.get("port", 5432).asUInt();
```

Comment:
We need to check overflow here?

Reply:
I think it's unnecessary. We didn't check it before, and we didn't check host format. Even if the number was wrong, one could easily find out reason from log.
If we check it, we must store it in uint type, and do convertion in every config construction.